core/deployment/src/main/java/io/quarkus/deployment/TestConfig.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.deployment;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+/**
+ * This is used currently only to suppress warnings about unknown properties
+ * when the user supplies something like: -Dquarkus.test.native-image-profile=someProfile
+ *
+ * TODO refactor code to actually use these values
+ */
+@ConfigRoot
+public class TestConfig {
+
+    /**
+     * Number of seconds to wait for the native image to built during testing
+     */
+    @ConfigItem(defaultValue = "300")
+    Integer nativeImageWaitTime;
```

Comment:
Hmm we probably should have made this a duration!

Reply:
Certainly, but it's unused currently, I just wanted to get rid of the warnings.
We should definitely make it a duration when we actually start using the class

Reply:
Fixed. @gsmet I force pushed to your branch for this one